code_challenges.rb

```diff
@@ -2,33 +2,135 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in an array of words and a letter and returns all the words that contain that particular letter.
-
+# 
+# # so I was having a very difficut time swicthing my mind from JS to ruby
+def find_o (array)
+    (array).select do |value|
+        value.include?'o'
+    end
+end
+# # I had to look up an example, for some reason I was thinging that it was going to be super easy to create the 
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
-letter_o = 'o'
+# letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
+p find_o (beverages_array)
+
+
+
+
+def find_t (array)
+    (array).select do |value|
+        value.include?'t'
+    end
+end
```

Comment:
Is there a way you could combine these? So that you don't have 'O' or 't' in your method? Remember we don't want to call on specific values in the methods/ functions. Just like we talked about last week